<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamMappingMOD1</class>
 <widget class="QDialog" name="ParamMappingMOD1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter Mapping</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>318</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnit">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>270</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dB</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>80</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Effect Block</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboEffectBlock"/>
   <widget class="QLabel" name="label_8">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parameter</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboParameter"/>
  </widget>
  <widget class="QLabel" name="lblMaxValue_2">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Max Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaxValue">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>240</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinValue_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Min Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>20</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select AXE-FX parameter</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinValue">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>240</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUnit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Unit</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cboEffectBlock</tabstop>
  <tabstop>cboParameter</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParamMappingMOD1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
